assets/components/prometheus-operator/manifests/crds/crd-alertmanager.yaml

```diff
@@ -5,6 +5,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.2.4
+    helm.sh/hook: crd-install
```

Comment:
Do you know why upstream is missing that?

Reply:
So helm chart upstream is not missing it. These CRDs are copied from the prometheus-operator repo where they provide just the yaml that can be used with `kubectl`. Hence they are missing there. This extra commit makes sure that we don't miss it and can find it in the history.

Reply:
Ah, make sense :+1: